features/networking/operator.feature

```diff
@@ -13,3 +13,167 @@ Feature: Operator related networking scenarios
+    """
+    When I run the :get admin command with:
+      | resource      | clusteroperators                                                  |
+      | resource_name | network                                                           |
+      | o             | jsonpath='{.status.conditions[?(.type == "Failing")].status}'     |
+    Then the step should succeed
+    And the output should contain "True"
+    When I run the :get admin command with:
+      | resource      | clusteroperators                                                  |
+      | resource_name | network                                                           |
+      | o             | jsonpath='{.status.conditions[?(.type == "Available")].status}'   |
+    Then the step should succeed
+    And the output should contain "True"
+    """
+    # Change the network.config.openshift.io back
+    When I run the :patch admin command with:
+      | resource      | network.config.openshift.io                       |
+      | resource_name | cluster                                           |
+      | p             | {"spec":{"networkType":"<%= cb.network_type %>"}} |
+      | type          | merge                                             |
+    Then the step should succeed
+    # Check that the operator status
+    Given I wait up to 20 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource      | clusteroperators                                                  |
+      | resource_name | network                                                           |
+      | o             | jsonpath='{.status.conditions[?(.type == "Failing")].status}'     |
+    Then the step should succeed
+    And the output should contain "False"
+    When I run the :get admin command with:
+      | resource      | clusteroperators                                                  |
+      | resource_name | network                                                           |
+      | o             | jsonpath='{.status.conditions[?(.type == "Available")].status}'   |
+    Then the step should succeed
+    And the output should contain "True"
+    """
+
+  # @author bmeng@redhat.com
+  # @case_id OCP-22202
+  @admin
+  @destructive
+  Scenario: The clusteroperator should be able to reflect the realtime status of the network when a new node added
+    Given the master version > "3.11"
+    # Check that the operator is not progressing
+    Given the expression should be true> cluster_operator('network').condition(type: 'Progressing')['status'] == "False"
+
+    # Record the original machine replica and scale it up to number +1
+    When I run the :get admin command with:
+      | resource | machineset                         |
+      | n        | openshift-machine-api              |
+      | template | {{(index .items 0).metadata.name}} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :machineset clipboard
```

Comment:
`:stdout` would be safer. Even better write a `MachineSet` class and use programmatic access.